azurerm/provider.go

```diff
@@ -63,6 +64,16 @@ func Provider() terraform.ResourceProvider {
 				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("ARM_SKIP_PROVIDER_REGISTRATION", false),
 			},
+			"use_msi": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("ARM_USE_MSI", false),
```

Comment:
Change to "USE_MSI".

Reply:
given all the other Environment Variables are prefixed with `ARM` - we should leave this as `ARM_USE_MSI` for consistency (we may change this for `AZURE_USE_MSI` in future, but it should be consistent for now)

Reply:
Agree.